python/pylibcudf/pylibcudf/libcudf/aggregation.pxd

```diff
@@ -170,3 +174,66 @@ cdef extern from "cudf/aggregation.hpp" namespace "cudf" nogil:
         null_policy null_handling,
         null_order null_precedence,
         rank_percentage percentage) except +libcudf_exception_handler
+
+cdef extern from "cudf/detail/aggregation/aggregation.hpp" \
+        namespace "cudf::detail" nogil:
```

Comment:
Hmm we don't expose pretty much any cudf detail APIs to pylibcudf, and I don't want to start here. Can we open an issue about these? If these are attributes that are absolutely necessary to reconstruct the serialized types, then we should discuss exposing them publicly in libcudf.